src/components/Company/CompanyName/CorrectNameOptions.vue

```diff
@@ -67,23 +80,28 @@ export default class CorrectNameOptions extends Vue {
   // local properties
   private displayedOptions: Array<CorrectNameOptionIF> = []
   private panel = null as number
```

Comment:
I'm not sure what this resolves to. Normally for a "non-value" for a number you'd use NaN.

In this case it may be falsey but that's a dangerous check because 0 is also falsey but is a valid number!

See https://www.w3schools.com/jsref/jsref_isnan.asp

Reply:
kk i'll look at this!

Reply:
Yea, i'm not too sure about this one.

Reply:
Oh, hehe, looks like I used "null" for "no panel" in Filings UI.

The Vuetify docs just say this is of type "any" :/

Well, leave it for now then. Just be aware not to check for "if (panel)".